### PR TITLE
Update certificates and add documentation.

### PR DESCRIPTION
**Pull request by dbudziwojskiNR**
## Type of change

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] New feature / enhancement (non-breaking change which adds functionality)
- [ ] Security fix
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] Add changelog entry following the [contributing guide](../CONTRIBUTING.md#pull-requests)
- [ ] Documentation has been updated
- [ ] This change requires changes in testing:
  - [x] unit tests
  - [ ] E2E tests

### CODE REVIEW

File: .github/workflows/push_pr.yaml [hunk 1]
```diff
@@ -138,3 +138,5 @@ jobs:
       - uses: actions/checkout@v4
       - name: Codespell test
         uses: codespell-project/actions-codespell@master
+        with:
+          ignore_words_list: enver,NotIn
```
Comment (juanjjaramillo) [inline .github/workflows/push_pr.yaml:142]: Awesome, Codespell issue fixed!

File: src/integration/sink/testdata/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+### Generate the Root CA Certificate:
+
+Copy the root CA generated by `mkcert`.
+
+```bash
+cd "`mkcert -CAROOT`"
+cat rootCA.pem
+```
+
+Replace the contents of `rootCA.pem` with the contents of `rootCA.pem`. Do **NOT** copy the key file.
```
Comment (juanjjaramillo) [inline src/integration/sink/testdata/README.md:16]: Can you please rephrase this sentence? It sounds you are replacing contents of a file with its own content, which is a noop. Also, can you explain why not simply copying the key file? There must be a good reason for that

**Review by juanjjaramillo — APPROVED** [review]

File: src/integration/sink/testdata/README.md (hunk 2 above)
Comment (dbudziwojskiNR) [reply src/integration/sink/testdata/README.md:16]: I see where the confusion can arise. I'll reword it to use copy instead of replace. Regarding the key file, the engineer needs to copy the CA cert, not the key. The CA key is the private key used to sign certs so committing that would would be bad, hence the warning. I'll clarify this.

Comment (dbudziwojskiNR) [reply src/integration/sink/testdata/README.md:16]: Thoughts on this: Copy `rootCA.pem` into `src/integration/sink/testdata` and rename it to `rootCA.pem` replacing the expired cert. Do **NOT** copy `rootCA-key.pem` file.

Comment (juanjjaramillo) [reply src/integration/sink/testdata/README.md:16]: So where do you store the key?
